graph/index.html

```diff
@@ -65,5 +65,21 @@
 <script src="//cdn.rawgit.com/bassjobsen/Bootstrap-3-Typeahead/master/bootstrap3-typeahead.min.js"></script>
 <script src="main.js"></script>
 <script>main();</script>
+<!-- Piwik -->
+<script type="text/javascript">
+  var _paq = _paq || [];
+  _paq.push(["setDomains", ["*.openfisca.fr","*.legislation.openfisca.fr","*.ui.openfisca.fr","*.www.openfisca.fr"]]);
+  _paq.push(['trackPageView']);
+  _paq.push(['enableLinkTracking']);
+  (function() {
+    var u="//stats.data.gouv.fr/";
```

Comment:
Please enforce `HTTPS`. Schema-less is deprecated, and we intend to only serve over HTTPS anyway.